ajax/libs/libphonenumber-js/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "libphonenumber-js",
+  "description": "A simpler (and smaller) rewrite of Google Android's popular libphonenumber library",
+  "version": "0.2.3",
+  "author": {
+    "name": "Halt Hammerzeit",
+    "email": "halt.hammerzeit.at@gmail.com"
+  },
+  "filename": "libphonenumber-js.js",
```

Comment:
Please modified `libphonenumber-.js` to `libphonenumber-js.min.js` in `filename` field, thanks!